pkgs/top-level/all-packages.nix

```diff
@@ -9612,6 +9612,7 @@ in
   ### DEVELOPMENT / INTERPRETERS
 
   acl2 = callPackage ../development/interpreters/acl2 { };
+  acl2-minimal = callPackage ../development/interpreters/acl2 { certifyBooks = false; };
```

Comment:
Are using this? Otherwise I would propose to not build this at all and just have the option so that someone can use `acl2.override ({ certifyBooks = false; })`

Reply:
The current situation is that `acl2` has no books but is cached at `cache.nixos.org`. After this PR, `acl2-minimal` will be what `acl2` used to be, and `acl2` will now have all the books compiled and certified etc., but will not be cached at `cache.nixos.org` and will be unavailable to people who don't want to turn on `allowUnfree`.

So I was concerned that if I didn't include `acl2-minimal` here, this PR could be considered a step backwards from the status quo, because while you could indeed do `acl2.override ({ certifyBooks = false; })` to get the old package, it would no longer be cached at `cache.nixos.org` since it would have no attribute in `all-packages.nix`. So for people who wanted ACL2 without the books, this would be strictly worse than before.

But no, personally I would not use this `acl2-minimal` package.

Reply:
I would leave the decision to @7c6f434c if he wants to maintain to separate versions